pkg/eventemitter/eventemitter.go

```diff
@@ -65,33 +68,51 @@ func (ee eventEmitter) EmitEvent(object runtime.Object, eventType, reason, msg s
 }
 
 func (ee eventEmitter) EmitProgressingForConfig() {
-	config := ee.getConfigForEmitter()
+	config, err := ee.getConfigForEmitter()
+	if err != nil {
+		log.Printf(GetErrorMessage+"%v", err)
```

Comment:
I think your already concatenated `GetErrorMessage` on `getConfigForEmitter()`
this error log should read something like this:
`log.Printf("Failed to emit event Progressing for config. err = %v", err)`

Reply:
oh right. damm.
fixed